library/src/main/java/nl/surfnet/nonweb/sso/SSOServiceActivity.java

```diff
@@ -62,29 +48,50 @@ public void onCreate(Bundle savedInstanceState) {
         if (BuildConfig.DEBUG) {
             Log.d(TAG, "Starting task to retrieve token.");
         }
-        new RequestTokenTask(this, _consumerId).execute();
+        new RequestTokenTask(this, //
```

Comment:
I find it harder to read this way, I guess it is subjective :)
You don't have to fix this, was just a note